execution_workflows/QAPA/more_samplesheets/samplesheet_single.csv

```diff
@@ -1,7 +1,7 @@
-sample,fastq1,fastq2,bam,bai,gff,fasta,bed,mart_export
-siControl_R1,/home/wanyk/Downloads/apa/data/fastqs/SRR11918577.fastq.gz,,/home/wanyk/Downloads/apa/results/star/siControl_R1.markdup.sorted.bam,/home/wanyk/Downloads/apa/results/star/siControl_R1.markdup.sorted.bam.bai,/home/wanyk/Downloads/apa/data/genome/gencode.v37.annotation.gff3,/home/wanyk/Downloads/apa/mm39.fa,/home/wanyk/Downloads/apa/qapa_3utrs.gencode_VM22.mm10.bed,/home/wanyk/Downloads/apa/mart_export.txt
-siControl_R2,/home/wanyk/Downloads/apa/data/fastqs/SRR11918578.fastq.gz,,/home/wanyk/Downloads/apa/results/star/siControl_R2.markdup.sorted.bam,/home/wanyk/Downloads/apa/results/star/siControl_R2.markdup.sorted.bam.bai,/home/wanyk/Downloads/apa/data/genome/gencode.v37.annotation.gff3,/home/wanyk/Downloads/apa/mm39.fa,/home/wanyk/Downloads/apa/qapa_3utrs.gencode_VM22.mm10.bed,/home/wanyk/Downloads/apa/mart_export.txt
-siSrsf3_R1,/home/wanyk/Downloads/apa/data/fastqs/SRR11918579.fastq.gz,,/home/wanyk/Downloads/apa/results/star/siSrsf3_R1.markdup.sorted.bam,/home/wanyk/Downloads/apa/results/star/siSrsf3_R1.markdup.sorted.bam.bai,/home/wanyk/Downloads/apa/data/genome/gencode.v37.annotation.gff3,/home/wanyk/Downloads/apa/mm39.fa,/home/wanyk/Downloads/apa/qapa_3utrs.gencode_VM22.mm10.bed,/home/wanyk/Downloads/apa/mart_export.txt
-siSrsf3_R2,/home/wanyk/Downloads/apa/data/fastqs/SRR11918580.fastq.gz,,/home/wanyk/Downloads/apa/results/star/siSrsf3_R2.markdup.sorted.bam,/home/wanyk/Downloads/apa/results/star/siSrsf3_R2.markdup.sorted.bam.bai,/home/wanyk/Downloads/apa/data/genome/gencode.v37.annotation.gff3,/home/wanyk/Downloads/apa/mm39.fa,/home/wanyk/Downloads/apa/qapa_3utrs.gencode_VM22.mm10.bed,/home/wanyk/Downloads/apa/mart_export.txt
-siSrsf7_R1,/home/wanyk/Downloads/apa/data/fastqs/SRR11918581.fastq.gz,,/home/wanyk/Downloads/apa/results/star/siSrsf7_R1.markdup.sorted.bam,/home/wanyk/Downloads/apa/results/star/siSrsf7_R1.markdup.sorted.bam.bai,/home/wanyk/Downloads/apa/data/genome/gencode.v37.annotation.gff3,/home/wanyk/Downloads/apa/mm39.fa,/home/wanyk/Downloads/apa/qapa_3utrs.gencode_VM22.mm10.bed,/home/wanyk/Downloads/apa/mart_export.txt
-siSrsf7_R2,/home/wanyk/Downloads/apa/data/fastqs/SRR11918582.fastq.gz,,/home/wanyk/Downloads/apa/results/star/siSrsf7_R2.markdup.sorted.bam,/home/wanyk/Downloads/apa/results/star/siSrsf7_R2.markdup.sorted.bam.bai,/home/wanyk/Downloads/apa/data/genome/gencode.v37.annotation.gff3,/home/wanyk/Downloads/apa/mm39.fa,/home/wanyk/Downloads/apa/qapa_3utrs.gencode_VM22.mm10.bed,/home/wanyk/Downloads/apa/mart_export.txt
+sample,fastq1,fastq2
```

Comment:
see above: probably best to remove these sample tables. Alternatively, mention these files with corresponding genome in the readme.

Reply:
I have removed the `more_samplesheets` directory